### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AothmanVeridium**
I fixed one misspelling and correct the role of ASA, the matching is still done within the UIDAI CIDR. There is the Aadhaar authentication server that its response will be passed to AUA again via the ASA.

### CODE REVIEW

File: concepts/0231-biometric-service-provider/README.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ as the most common use case:
 ![bsp_aadhaar](images/bsp_aadhaar.png)
 
 Authentication Service Agents (ASAs) are licensed by the Government of
-India to connect via secure channels to fetch IBVs from the Unique
-Identification Authority of India (UIDAI) data centre and match them
+India to pass the verification request via secure channels to the Unique
+Identification Authority of India (UIDAI) data centre where IBVs are retireved and matched
```
Comment (llorllale) [inline concepts/0231-biometric-service-provider/README.md]: *retrieved

**Comment by kdenhartog** [comment]
The DCO is failing because this commit isn't signed. For details on how to fix this, check the signing DCO commit section in the [Contributors guide](https://github.com/hyperledger/aries-rfcs/blob/e6c96a872bbdc9b111afb64120ec88b59a4991c2/contributing.md#signing-off-commits-dco).

**Comment by kdenhartog** [comment]
I'm not sure what's happening with this git log and it seems like it's caused some conflicts. @AothmanVeridium did you do a `git rebase`?

**Comment by dhh1128** [comment]
We accidentally got some other commit history sprinkled in. I will resolve. Give me a few minutes.

**Comment by dhh1128** [comment]
This PR is superseded by PR #285 , which contains Asem's exact checkin minus all the extra commits that were accidentally added here. Since that PR is now merged with Asem's update applied, this PR can now be closed.

**Pull request closed without merging**